diff --git a/.gitignore b/.gitignore
index 93821ad..9bc0690 100644
--- a/.gitignore
+++ b/.gitignore
@@ -14,3 +14,5 @@
 !var/SymfonyRequirements.php
 /vendor/
 /web/bundles/
+/node_modules
+/web/builds/
diff --git a/app/Resources/views/default/homepage.html.twig b/app/Resources/views/default/homepage.html.twig
index 020b61c..c956439 100644
--- a/app/Resources/views/default/homepage.html.twig
+++ b/app/Resources/views/default/homepage.html.twig
@@ -3,7 +3,7 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script src="{{ asset('js/productApp.js') }}"></script>
+    <script src="{{ asset('builds/productApp.js') }}"></script>
 {% endblock %}
 
 {% block body %}
diff --git a/package.json b/package.json
index b8a30c9..c30bdc3 100644
--- a/package.json
+++ b/package.json
@@ -1,4 +1,7 @@
 {
   "name": "webpack-tutorial",
-  "version": "1.0.0"
+  "version": "1.0.0",
+  "devDependencies": {
+    "babel-cli": "^6.10.1"
+  }
 }
