diff --git a/app/Resources/views/default/homepage.html.twig b/app/Resources/views/default/homepage.html.twig
index d65b617..020b61c 100644
--- a/app/Resources/views/default/homepage.html.twig
+++ b/app/Resources/views/default/homepage.html.twig
@@ -1,5 +1,11 @@
 {% extends 'base.html.twig' %}
 
+{% block javascripts %}
+    {{ parent() }}
+
+    <script src="{{ asset('js/productApp.js') }}"></script>
+{% endblock %}
+
 {% block body %}
 <div class="hero-space">
     <div class="container">
diff --git a/web/js/productApp.js b/web/js/productApp.js
new file mode 100644
index 0000000..ab635b6
--- /dev/null
+++ b/web/js/productApp.js
@@ -0,0 +1,18 @@
+var products = [
+    'Sheer Shears',
+    'Wool Hauling Basket',
+    'After-Shear (Fresh Cut Grass)',
+    'After-Shear (Morning Dew)',
+    'Shear Comb',
+    'Shearly Conditioned'
+];
+
+var loopThroughProducts = function(callback) {
+    for (var i = 0, length = products.length; i < length; i++) {
+        callback(products[i]);
+    }
+};
+
+loopThroughProducts(function(product) {
+    console.log('Product: '+product);
+});
\ No newline at end of file
