diff --git a/web/js/productApp.js b/web/js/productApp.js
new file mode 100644
index 0000000..ab635b6
--- /dev/null
+++ b/web/js/productApp.js
@@ -0,0 +1,18 @@
+var products = [
+    'Sheer Shears',
+    'Wool Hauling Basket',
+    'After-Shear (Fresh Cut Grass)',
+    'After-Shear (Morning Dew)',
+    'Shear Comb',
+    'Shearly Conditioned'
+];
+
+var loopThroughProducts = function(callback) {
+    for (var i = 0, length = products.length; i < length; i++) {
+        callback(products[i]);
+    }
+};
+
+loopThroughProducts(function(product) {
+    console.log('Product: '+product);
+});
\ No newline at end of file
diff --git a/web/products.html b/web/products.html
new file mode 100644
index 0000000..1686abb
--- /dev/null
+++ b/web/products.html
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Products</title>
+</head>
+<body>
+    <h1>Products</h1>
+
+    <script src="js/productApp.js"></script>
+</body>
+</html>
\ No newline at end of file
