diff --git a/package.json b/package.json
index a80ccf3..c9d8c8e 100644
--- a/package.json
+++ b/package.json
@@ -7,6 +7,7 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.9.0",
     "css-loader": "^0.23.1",
+    "style-loader": "^0.13.1",
     "webpack": "^1.13.1",
     "webpack-dev-server": "^1.14.1"
   }
diff --git a/web/js/productApp.js b/web/js/productApp.js
index c7480f5..6603008 100644
--- a/web/js/productApp.js
+++ b/web/js/productApp.js
@@ -1,6 +1,5 @@
 import ProductCollection from './ProductCollection';
-import cssData from 'css!../css/productApp.css';
-console.log(cssData);
+import 'style!css!../css/productApp.css';
 
 var collection = new ProductCollection([
     'Sheer Shears',
