diff --git a/app/Resources/views/default/homepage.html.twig b/app/Resources/views/default/homepage.html.twig
index 363d561..c956439 100644
--- a/app/Resources/views/default/homepage.html.twig
+++ b/app/Resources/views/default/homepage.html.twig
@@ -3,7 +3,6 @@
 {% block javascripts %}
     {{ parent() }}
 
-    <script src="{{ asset('js/ProductCollection.js') }}"></script>
     <script src="{{ asset('builds/productApp.js') }}"></script>
 {% endblock %}
 
diff --git a/web/js/ProductCollection.js b/web/js/ProductCollection.js
index 7b3ef84..f528edf 100644
--- a/web/js/ProductCollection.js
+++ b/web/js/ProductCollection.js
@@ -12,3 +12,5 @@ class ProductCollection
         return this.products[i];
     }
 }
+
+export ProductCollection;
\ No newline at end of file
diff --git a/web/js/productApp.js b/web/js/productApp.js
index bafb672..707f228 100644
--- a/web/js/productApp.js
+++ b/web/js/productApp.js
@@ -1,3 +1,5 @@
+import ProductCollection from './ProductCollection';
+
 var collection = new ProductCollection([
     'Sheer Shears',
     'Wool Hauling Basket',
