diff --git a/web/js/ProductCollection.js b/web/js/ProductCollection.js
index 7b3ef84..f528edf 100644
--- a/web/js/ProductCollection.js
+++ b/web/js/ProductCollection.js
@@ -12,3 +12,5 @@ class ProductCollection
         return this.products[i];
     }
 }
+
+export ProductCollection;
\ No newline at end of file
diff --git a/web/js/productApp.js b/web/js/productApp.js
index bafb672..707f228 100644
--- a/web/js/productApp.js
+++ b/web/js/productApp.js
@@ -1,3 +1,5 @@
+import ProductCollection from './ProductCollection';
+
 var collection = new ProductCollection([
     'Sheer Shears',
     'Wool Hauling Basket',
diff --git a/web/products.html b/web/products.html
index ae5d1be..aee8f15 100644
--- a/web/products.html
+++ b/web/products.html
@@ -7,7 +7,6 @@
 <body>
     <h1>Products</h1>
 
-    <script src="js/ProductCollection.js"></script>
     <script src="js/compiled.js"></script>
 </body>
 </html>
\ No newline at end of file
