diff --git a/web/js/Components/ProductApp.js b/web/js/Components/ProductApp.js
new file mode 100644
index 0000000..0a0be89
--- /dev/null
+++ b/web/js/Components/ProductApp.js
@@ -0,0 +1,11 @@
+import React from 'react';
+
+var ProductApp = React.createClass({
+    render: function() {
+        return (
+            <h1>Yay!</h1>
+        )
+    }
+});
+
+module.exports = ProductApp;
diff --git a/web/js/productApp.js b/web/js/productApp.js
index f71c01c..9d89cb3 100644
--- a/web/js/productApp.js
+++ b/web/js/productApp.js
@@ -1,8 +1,8 @@
 import ProductCollection from './ProductCollection';
 import '../css/productApp.css';
-import _ from 'lodash';
 import React from 'react';
 import ReactDOM from 'react-dom';
+import ProductApp  from './Components/ProductApp';
 
 var collection = new ProductCollection([
     'Sheer Shears',
@@ -13,14 +13,6 @@ var collection = new ProductCollection([
     'Shearly Conditioned'
 ]);
 
-var ProductApp = React.createClass({
-    render: function() {
-        return (
-            <h1>Yay!</h1>
-        )
-    }
-});
-
 $(document).ready(function() {
     ReactDOM.render(
         <ProductApp/>,
